README.md

```diff
@@ -1 +1,187 @@
-# Bookstore
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="carbook.png" alt="logo" width="180"  height="auto" />
+  <br/>
+
+  <h3><b>Book Store</b></h3>
+  <p>this is a book selling app</p>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[BookStore\] ](#-BookStore-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Build](#build)
+    - [Run](#run)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [© Acknowledge ](#-acknowledge-)
+  - [⭐ Show your support ](#-show-your-support-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 [Book Store] <a name="about-project"></a>
+
+*[math-magician]* a Single Page WebApp (API based) that shows a BookStore
+<div align="center">
+
+  <img src="inprogress..." alt="Screenshoot" width="550"  height="auto" />
+  <br/>
+
+</div>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML" target="_blank" rel="noopener noreferrer">HTML</a></li>
+    <li><a href="https://es.reactjs.org/" target="_blank" rel="noopener noreferrer">React</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS" target="_blank" rel="noopener noreferrer">CSS</a></li>
+    <li><a href="https://redux.js.org/" target="_blank" rel="noopener noreferrer">Redux</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" target="_blank" rel="noopener noreferrer">Javascript</a></li>
+  </ul>
+
+
+### Key Features <a name="key-features"></a>
+
+- *DOM Manipulation for showing a BookStore.*
+- *you can watch diferents books.*
+- *Use React.*
+- *Use Redux.*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="in progres..." target="_blank" rel="noopener noreferrer">math-magician</a>
```

Comment:
[OPTIONAL]
- [ ] You may consider changing the text of the anchor tag as well. `Book store` instead of `Math magician`

Reply:
thanks i will do that